articles.md

```diff
@@ -23,7 +23,7 @@ tags: [node, wallet, security, privacy, nokyc]
 | Name                             |  Description                                                     |  Published                             |
 |----------------------------------|----------|-----------------------------------------------------------------|----------------------------------------|
 | [Blixt 101](/blixt)   |  Collaborative post with [Expatriotic](https://twitter.com/expatri0tic) on using Blixt wallet  |  05 Jul 2023            |
-| [Stack Duo 101](/stack-duo)   |  Guest post by [Bitlevi](https://twitter.com/Bit_levi) on using Stack Duo mobile wallet  |  16 May 2023            |
+| [Stack Duo 101](/stack-duo)   |  Guest post by [Bitlevi](https://twitter.com/Bit_levi) on using Stack Duo mobile wallet  | Updated 09 August 2023            |
```

Comment:
Can you update the month so the formatting matches?

Eg `August` > `Aug`.

I tried to do it myself but you didn't enabled 'Allow edits by maintainers'.

Cheers man